x-pack/plugins/siem/public/components/open_timeline/timelines_table/extended_columns.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as React from 'react';
+
+import { getEmptyValue } from '../../empty_value';
+
+import * as i18n from '../translations';
+
+/**
+ * Returns the expanded columns that are specific to the `All Timelines` view
+ * of the `Timelines` page
+ */
+export const getExtendedColumns = () => [
+  {
+    dataType: 'string',
+    field: 'updatedBy',
+    name: i18n.MODIFIED_BY,
+    render: (updatedBy: string) => (
+      <div data-test-subj="username">{updatedBy != null ? updatedBy : getEmptyValue()}</div>
+    ),
+    sortable: true,
+  },
+];
```

Comment:
Optional, you can use the `Columns` type we have constructed a while ago and re-write this as:

```ts
/**
 * Returns the expanded columns that are specific to the `All Timelines` view
 * of the `Timelines` page
 */
export const getExtendedColumns = (): [Columns<TimelineResult['updatedBy']>] => [
  {
    field: 'updatedBy',
    name: i18n.MODIFIED_BY,
    render: updatedBy => (
      <div data-test-subj="username">{defaultToEmptyTag(updatedBy)}</div>
    ),
    sortable: true,
  },
];
```

The type of Columns doesn't have `dataType` so if you are uncomfortable with `dataType: auto` you would have to add it this file:

```
x-pack/plugins/siem/public/components/load_more_table/index.tsx
```

Then you would get the correct type of `render` automatically when hovering:

<img width="675" alt="Screen Shot 2019-05-01 at 3 04 19 PM" src="https://user-images.githubusercontent.com/1151048/57042735-cb7a7980-6c22-11e9-8e0a-78a8d2a494ba.png">

If you don't feel like messing with that, at the least I suggest at least using the type directly from your `TimelineResult` so that they stay compatible if you update `TimelineResult` rather than re-typing:

```ts
  render: (updatedBy: TimelineResult['updatedBy']) => (
      <div data-test-subj="username">{defaultToEmptyTag(updatedBy)}</div>
    ),
``` 

Credit to @spong , for showing me that cool neat little trick to keep types more strict.